src/data-model/SerializedObject.ts

```diff
@@ -1,11 +1,13 @@
 import PropertyValueRelation from '@/data-model/PropertyValueRelation';
 import { Value } from '@/store/RootState';
+import ConditionRelation from './ConditionRelation';
 
 export default interface SerializedCondition {
 	propertyId: string;
 	propertyDataType: string | null;
 	propertyValueRelation: PropertyValueRelation;
 	value: string | Value;
 	subclasses: boolean;
+	conditionRelation: ConditionRelation | null;
```

Comment:
The logic in the code implies this condition relation is between this condition and the condition above (funnily enough, when implementing the component, I went with the exact opposite). This should be documented.

Reply:
You mean by adding a comment? Or documented how?

Reply:
Yes. Technically the relation doesn't belong to the query condition. It determines the relation between two. A comment in the QueryCondition would be good enough IMO

Reply:
I added some comments here

Reply:
Thanks. It's just it's below it and it should be usually above. Also maybe it should be in the RootState? here is mostly about sharing the link.

Reply:
done. :roll_eyes: